tests/test_n_queens.py

```diff
@@ -23,8 +23,9 @@
 
 project_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-class TestSmoke(unittest.TestCase):
-    def test_smoke(self):
+class TestNQueens(unittest.TestCase):
+    @unittest.skipIf(os.getenv('SKIP_INT_TESTS'), "Only run int tests w/ latest supported Python, on Linux.")
```

Comment:
I wouldn't go into (internal) details when we'll skip some integration test. Also, since the skip flags are controlled from our orb, we don't want to hard-code the configuration in _every_ example.

Reply:
Ah ok, I thought by saying "latest supported python", I was skipping internal details, but I guess that or the platform could change.. Will just say “Skipping integration test” here then.